benchmark_utils/config.py

```diff
@@ -1,8 +1,20 @@
 from pathlib import Path
+from socket import gethostname
 
-root = Path(
-    "/data/parietal/store3/work/pbarbara/datasets/fmralign_benchopt_data"
-)
+if gethostname().startswith("drago"):  # drago* machine
+    root = Path("/storage/store2/work/dfouchar/fmralign_benchopt_data/")
+    # path to memory
+    MEMORY = "/storage/store2/work/dfouchar/tmp"
+
+elif gethostname().startswith("mar"):  # margaret node
+    root = Path("/data/parietal/store2/work/dfouchar/fmralign_benchopt_data/")
+    # path to memory
+    MEMORY = "/data/parietal/store2/work/dfouchar/tmp"
```

Comment:
No way to find a user independent path ?

Reply:
Just edited the code of Pierre-Louis on this one, and I think the user actively using his path is better suited when working on a shared computing cluster, as we don't want access errors or unwanted files filing space where it should not